src/index.html

```diff
@@ -11,7 +11,228 @@
+              > Yes
+            </label>
+            <label for="nocats">
+              <input
+                type="radio"
+                name="cats"
+                id="nocats"
+              > No
+            </label>
+          </div>
+          <div class="input">
+            <label for="favcolor">
+              What is your favorite color?
+              <input
+                type="color"
+                name="color"
+                id="favcolor"
+              >
+            </label>
+          </div>
+          <div class="input">
+            <label for="bed">
+              What time do you go to bed?
+              <input
+                type="time"
+                name="time"
+                id="bed"
+                step="1"
+              >
+            </label>
+          </div>
+          <div class="input">
+            <label for="cars"
+            class="input">
+              What are your favorite brand of cars?
+              <select
+                name="brand"
+                id="cars"
+                multiple
+              >
+                <option value="BMW"
+                  name="BMW">BMW
+                </option>
+                <option value="Audi"
+                  name="Audi">Audi
+                </option>
+                <option value="Lada"
+                  name="Lada">Lada
+                </option>
+              </select>
+            </label>
+          </div>
+          <div class="last">
+            <label for="workrate"
+            class="last">
+              How do you rate our work?
+              <input
+                type="range"
+                name="rate"
+                id="workrate"
+                value="0"
+              >
+            </label>
+          </div>
+        </fieldset>
+      </div>
+      <div class="field">
+        <fieldset>
+          <legend>Additional info:</legend>
+          <div class="input">
+            <label for="comments">
+              Comments:
+              <textarea
+                name="commentme"
+                id="comments"
+                cols="20"
+                rows="2"
+              >
+              </textarea>
+            </label>
+          </div>
+          <div class="last">
+            <label for="rec"
+            class="last">
+              Would you recommend us?
+              <select
+                name="recommend"
+                id="rec"
+                class="last"
+              >
+                <option
+                  value="yes"
+                  name="yes"
+                >yes
+                </option>
+                <option
+                  value="no"
+                  name="no"
+                >no
+                </option>
+              </select>
+            </label>
+          </div>
+        </fieldset>
+      </div>
+      <div>
+        <button
+          type="submit"
+          name="submit1"
+          class="button"
+        >Submit
```

Comment:
Name here is useless.
```suggestion
        <button
          type="submit"
          class="button"
        >
        Submit
```